ansible-scylla-node/tasks/adjust_keyspace_replication.yml

```diff
@@ -24,9 +24,20 @@
   loop: "{{ groups['scylla'] }}"
   run_once: true
 
+- set_fact:
+    scylla_admin_username: "{{ item.split('=')[0] }}"
+    scylla_admin_password: "{{ item.split('=')[1] }}"
+  when: "'cql_credentials' in groups and item.split('=')[0] == scylla_admin_default_user | default('scylla_admin')"
```

Comment:
This looks wrong - there is a native way to read values defined in inventory instead of parsing them as plain text.

Reply:
Did you read this page: https://docs.ansible.com/ansible/latest/collections/ansible/builtin/ini_inventory.html ?

This page states that defining `key=value` is definitely supported and not only in `:vars` sections.